pkgs/applications/networking/cluster/hadoop/default.nix

```diff
@@ -34,7 +34,7 @@ let
           outputHash = dependencies-sha256;
         };
 
-        nativeBuildInputs = [ maven cmake pkgconfig ];
+        nativeBuildInputs = [ maven cmake pkgconfig jdk ];
```

Comment:
this probably needless, `${pkgs.maven}/bin/mvn` wrapper sets `$JAVA_HOME` to `pkgs.jdk`

Reply:
Yeah, I don't think it's needed - I was just thinking explicit would be somehow nice.

Reply:
It created a false feeling that changing (or specifying) `jdk` would have an effect.